### PR TITLE
Clarify Pages caching behavior

### PR DESCRIPTION
**Pull request by Albertozhao**
This tells Pages users how to ensure the latest build gets served. Sometimes pre-existing cache settings/Page Rules can mess with the default caching behavior for Pages, leading to stale assets being served.

### CODE REVIEW

**Review by jkup — APPROVED** [review]
Looks good to me! As a customer, I'm always curious why my cache % is lower than I'd like it to be!

File: products/pages/src/content/platform/serving-pages.md [hunk 1]
```diff
@@ -22,4 +22,12 @@ If your project does not include a top-level `404.html` file, Pages assumes that
...
+Pages includes good caching defaults. That means that every time you deploy an asset to Pages, it remains cached on the Cloudflare CDN until your next deploy. 
+
+<Aside type="note" header="Purging the cache">
+
+If Page Rules or other cache settings are used on your custom domain, that may lead to stale assets being served after a new build. You can resolve this quickly by clicking <a href="https://developers.cloudflare.com/cache/how-to/purge-cache#purge-everything">Purge Everything</a> in the "Caching" tab of your zone to ensure the latest build gets served."
+
+</Aside>
+
+As much as possible, Pages sets `ETag` and `If-None-Match` headers to allow clients to also cache content in their browsers — for more details on these behaviors, refer to the MDN [HTTP Caching page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching). Pages will also serve Gzip and Brotli responses whenever possible.
```
Comment (rita3ko) [inline products/pages/src/content/platform/serving-pages.md]: we should be able to provide more clarity than "As much as possible". Can we say anything more specific about why we wouldn't? And why that's the approach we take to caching.

Comment (jkup) [reply products/pages/src/content/platform/serving-pages.md]: That's a good point. Also, for the "Gzip and Brotli" section - I think that's configureable via the Dashboard? Is it Gzip by default and you can upgrade to Brotli on the Speed page. Or is Brotli a Pro only feature?

Comment (Albertozhao) [reply products/pages/src/content/platform/serving-pages.md]: @rita3ko totally, we definitely can elaborate more, but just going to double check with Cina on how much to publicly disclose once he's back from break. 

@jkup that section is referring to the pre-processing of assets we do during the build step, not the zone settings that allow other compression options. But that definitely should be explained more clearly. :)

File: products/pages/src/content/platform/serving-pages.md [hunk 2]
```diff
@@ -22,4 +22,12 @@ If your project does not include a top-level `404.html` file, Pages assumes that
 
 ## Caching and performance
 
-Pages includes good caching defaults. That means that every time you deploy an asset to Pages, it remains cached on the Cloudflare CDN until your next deploy. As much as possible, Pages sets `ETag` and `If-None-Match` headers to allow clients to also cache content in their browsers — for more details on these behaviors, refer to the MDN [HTTP Caching page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching). Pages will also serve Gzip and Brotli responses whenever possible.
+Pages includes good caching defaults. That means that every time you deploy an asset to Pages, it remains cached on the Cloudflare CDN until your next deploy. 
+
+<Aside type="note" header="Purging the cache">
+
+If Page Rules or other cache settings are used on your custom domain, that may lead to stale assets being served after a new build. You can resolve this quickly by clicking <a href="https://developers.cloudflare.com/cache/how-to/purge-cache#purge-everything">Purge Everything</a> in the "Caching" tab of your zone to ensure the latest build gets served."
```
Comment (deadlypants1973) [inline products/pages/src/content/platform/serving-pages.md]:
```suggestion
If Page Rules or other cache settings are used on your custom domain, that may lead to stale assets being served after a new build. You can resolve this by selecting **Caching** > **Configuration** > <a href="https://developers.cloudflare.com/cache/how-to/purge-cache#purge-everything">**Purge Everything**</a> in the dashboard to ensure the latest build gets served."
```

File: products/pages/src/content/platform/serving-pages.md [hunk 3]
```diff
@@ -22,4 +22,12 @@ If your project does not include a top-level `404.html` file, Pages assumes that
 
 ## Caching and performance
 
-Pages includes good caching defaults. That means that every time you deploy an asset to Pages, it remains cached on the Cloudflare CDN until your next deploy. As much as possible, Pages sets `ETag` and `If-None-Match` headers to allow clients to also cache content in their browsers — for more details on these behaviors, refer to the MDN [HTTP Caching page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching). Pages will also serve Gzip and Brotli responses whenever possible.
+Pages includes good caching defaults. That means that every time you deploy an asset to Pages, it remains cached on the Cloudflare CDN until your next deploy.
```
Comment (deadlypants1973) [inline products/pages/src/content/platform/serving-pages.md]:
```suggestion
Pages includes good caching defaults. Every time you deploy an asset to Pages, the asset remains cached on the Cloudflare CDN until your next deployment. 
```

**Review by lukeed — COMMENTED** [review]
I've seen the question of Pages caching & Range requests (`206`) come up a few times. Maybe that's worth mentioning here too?

File: products/pages/src/content/platform/serving-pages.md [hunk 4]
```diff
@@ -32,6 +32,6 @@ If Page Rules or other cache settings are used on your custom domain, that may l
 
 For browser caching, Pages always sends `Etag` headers for `200 OK` responses, which the browser then returns in an `If-None-Match` header on subsequent requests for that asset. Pages compares the `If-None-Match` header from the request with the `Etag` it's planning to send, and if they match, Pages instead responds with a `304 Not Modified` that tells the browser it's safe to use what is stored in local cache.
 
-Pages currently returns `200` responses for HTTP range requests, however, The team is working on adding support for `206` and other responses.
+Pages currently returns `200` responses for HTTP range requests, however, the team is working on adding support for `206` and other responses.
```
Comment (lukeed) [inline products/pages/src/content/platform/serving-pages.md]:
```suggestion
Pages currently returns `200` responses for HTTP range requests; however, the team is working on adding spec-compliant `206` partial responses.
```

**Review by lukeed — APPROVED** [review]

File: products/pages/src/content/platform/serving-pages.md [hunk 5]
```diff
@@ -22,4 +22,16 @@ If your project does not include a top-level `404.html` file, Pages assumes that
 
 ## Caching and performance
 
-Pages includes good caching defaults. That means that every time you deploy an asset to Pages, it remains cached on the Cloudflare CDN until your next deploy. As much as possible, Pages sets `ETag` and `If-None-Match` headers to allow clients to also cache content in their browsers — for more details on these behaviors, refer to the MDN [HTTP Caching page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching). Pages will also serve Gzip and Brotli responses whenever possible.
+Pages includes highly performant caching defaults, so you should avoid setting Page Rules or custom caching on your site. Every time you deploy an asset to Pages, the asset remains cached on the Cloudflare CDN until your next deployment.
```
Comment (rita3ko) [inline products/pages/src/content/platform/serving-pages.md]:
```suggestion
Pages comes with built in caching defaults that are optimized for caching as much as possible, while providing the most up to date content. Every time you deploy an asset to Pages, the asset remains cached on the Cloudflare CDN until your next deployment. Therefore, you should avoid setting Page Rules or custom caching on your site. 
```

**Review by rita3ko — CHANGES_REQUESTED** [review]

**Comment by johtso** [comment]
Is there any update on support for range requests? Does Cloudflare Pages currently not support range requests at all?

**Comment by rickybassom** [comment]
I'm also interested in when range requests will be supported. Is there any news on this?

**Pull request merged**